src/store/callViewStore.js

```diff
@@ -15,6 +15,7 @@ const state = {
 	isViewerOverlay: false,
 	isGrid: false,
 	isStripeOpen: true,
+	isEmptyCallView: false,
```

Comment:
Shouldn't it be true by default? We don't have any participants models by the time we just have joined the call, so don't need to immediately rewrite it

Reply:
And set back to true after leaving the call (to prepare app for the next call)